### PR TITLE
[CoW] Migrate Airdrop Claims

### PR DESCRIPTION
**Pull request by bh2smith**
Should be a straight forward migration, but I changed the style of the query (personal preference).

PoC Query is here - working as intended: https://dune.com/queries/2731802

tagging @hildobby as reviewer (since this was your table to begin with)

cc @gentrexha

### CODE REVIEW

File: models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql [hunk 1]
```diff
@@ -2,45 +2,48 @@
...
         unique_key = ['recipient', 'tx_hash', 'evt_index'],
         post_hook='{{ expose_spells(\'["ethereum"]\',
                                 "project",
                                 "cow_protocol",
-                                \'["hildobby"]\') }}'
+                                \'["hildobby", "bh2smith"]\') }}'
     )
 }}
 
-{% set cow_token_address = '0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab' %}
```
Comment (bh2smith) [inline models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql:16]: Wasn't sure if this was still a thing in dune sql and didn't bother risking it. Can put it back if necessary.

File: models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql [hunk 2]
```diff
@@ -2,45 +2,48 @@
     config(
         schema = 'cow_protocol_ethereum',
         alias='airdrop_claims',
```
Comment (bh2smith) [inline models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql:4]:
```suggestion
        alias=alias('airdrop_claims'),
```

File: models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql [hunk 3]
```diff
@@ -2,45 +2,48 @@
...
+        tags=['dunesql'],
         materialized = 'table',
         file_format = 'delta',
         tags=['static'],
```
Comment (bh2smith) [inline models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql:8]:
```suggestion
        materialized = 'table',
        file_format = 'delta',
        tags=['static', 'dunesql'],
```

File: models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql (hunk 1 above)
Comment (jeff-dude) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql:16]: this is still a thing. i believe the setup is:
`{% set cow_token_address = '0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab' %}`
`AND contract_address={{cow_token_address}}`

can you try this? maybe a quick compile and see if print the address without single quotes? then push commit back here?

Comment (jeff-dude) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql:16]: join condition too:
`AND pu.contract_address={{cow_token_address}}`

**Comment by bh2smith** [comment]
I believe this is superseded by #3757, but not entirely sure.

**Review by jeff-dude — APPROVED** [review]
went ahead and helped finalize this one, should be all set

**Comment by gentrexha** [comment]
> went ahead and helped finalize this one, should be all set

Thanks Jeff!

**Pull request merged**